vm/qemu/qemu.go

```diff
@@ -171,7 +176,7 @@ var archConfigs = map[string]*archConfig{
 		Qemu:      "qemu-system-s390x",
 		TargetDir: "/",
 		QemuArgs:  "-M s390-ccw-virtio -cpu max,zpci=on",
-		NicModel:  ",model=virtio",
+		NetDev:    "virtio-net-pci",
```

Comment:
@eaibmz I assume it's the same as before, just the new way to specify the network config. Is something breaks regarding network, keep in mind this change :)